lib/plugins/create/templates/aws-golang/serverless.yaml

```diff
@@ -0,0 +1,51 @@
+# Welcome to Serverless!
+#
+# This file is the main config file for your service.
+# It's very minimal at this point and uses default values.
+# You can always add more config options for more control.
+# We've included some commented out config examples here.
+# Just uncomment any of them to get that config option.
+#
+# For full config options, check the docs:
+#    v1.docs.serverless.com
+#
+# Happy Coding!
+
+service: aws-golang
+provider:
+  name: aws
+  runtime: nodejs4.3
+
+# you can overwrite defaults here
+# defaults:
+#   stage: dev
+#   region: us-east-1
+
+# you can add packaging information here
+package:
+  # include:
+  #   - include-me.js
+  exclude:
+    - hello.go
+  # artifact: my-service-code.zip
+
+functions:
+  hello:
+    handler: handler.hello
+
+#   you can add any of the following events
+#   events:
+#     - http:
+#       path: users/create
```

Comment:
This was changed recently (we've added two more indentations for the nested events).